packages/webview_flutter/webview_flutter_wkwebview/darwin/webview_flutter_wkwebview/Sources/webview_flutter_wkwebview/WebKitLibrary.g.swift

```diff
@@ -4344,7 +4344,7 @@ final class PigeonApiWKNavigationDelegate: PigeonApiProtocolWKNavigationDelegate
         let message: String? = nilOrValue(listResponse[1])
         let details: String? = nilOrValue(listResponse[2])
         completion(.failure(PigeonError(code: code, message: message, details: details)))
-      } else if listResponse[0] == nil {
+      } else if listResponse[0] == nil || firstElement is NSNull {
```

Comment:
Also for reference in case you plan to contribute to `webview_flutter` in the future: Per the comment at the top of this file, it cannot be edited like this.

> // Autogenerated from Pigeon (v22.7.2), do not edit directly.